src/rdb.c

```diff
@@ -2909,13 +2909,11 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
     buf[9] = '\0';
     if (memcmp(buf,"REDIS",5) != 0) {
         serverLog(LL_WARNING,"Wrong signature trying to load DB from file");
-        errno = EINVAL;
```

Comment:
just double checking on when these were introduced: 6d61e5bf5b3bdbdfa3032b2009120c3cceb64607
i think they weren't really needed, and our explicit return value for a missing file is a better solution (considering there are many other cases for this function to return with god knows which errno....

Reply:
yes, agree, so i dropped it, relying on errno is really unreliable...